protocols/gossipsub/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.44.3 - unreleased
```

Comment:
Can you bump the version in the Cargo.toml too please?

Reply:
Done